test/phabricator_tests/test_phabricator.py

```diff
@@ -0,0 +1,363 @@
+import unittest
+try:
+    # Python 3 >
+    from unittest.mock import patch
+except:
+    from mock import patch
+try:
+    # Python 3.3 >
+    from unittest.mock import MagicMock
+except:
+    from mock import MagicMock
+from unittest import TestCase
+import datetime
+
+from reviewrot.phabricatorstack import PhabricatorService, PhabricatorReview
+from reviewrot.basereview import BaseService, BaseReview, LastComment
+from mock_phabricator import MockPhabricator
```

Comment:
I have problem with this import.
    `ModuleNotFoundError: No module named 'mock_phabricator'`

Solved it by changing the import a bit
```python
from .mock_phabricator import MockPhabricator
```

Reply:
How are you running the test cases? Just tried changing it and it broke my local version.

Reply:
Closing this one. If they are fine at your side, thats good. How the import should be will be decided in the future. There is a plan to create a tox configuration which would execute the tests. It depends if the tox will require the change I posted.